### PR TITLE
all: fix url formatting in config docs

### LINKED ISSUE #3879: [cisco_umbrella] Description of proxy setting results as hyperlink
It looks like that the 's' of 'https' of the proxy setting results into a link.

https://github.com/elastic/integrations/blob/26aa97053b98eb9005ca7e8ae66644f0e1161739/packages/cisco_umbrella/data_stream/log/manifest.yml#L130

image

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

Fixes rendering of the proxy config docs. See screen shot below.

## Checklist

- [ ] ~~I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.~~
- [ ] ~~I have verified that all data streams collect metrics or logs.~~
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] ~~I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).~~

## Author's Checklist

- [ ]

## Related issues

- Closes #3879

## Screenshots

Screen Shot 2022-07-28 at 19 06 01

### CODE REVIEW

File: packages/cisco_umbrella/data_stream/log/manifest.yml [hunk 1]
```diff
@@ -127,7 +127,7 @@ streams:
         multi: false
         required: false
         show_user: false
-        description: URL to proxy connections in the form of http[s]://<user>:<password>@<server name/ip>:<port>
+        description: URL to proxy connections in the form of http\[s\]://<user>:<password>@<server name/ip>:<port>
```
Comment (legoguy1000) [inline packages/cisco_umbrella/data_stream/log/manifest.yml:130]: Is it only this integration that's affected?? I copied and pasted this from another integration so if is rendering as a link here, I suspect all the other integrations with the proxy config is affected.

Comment (efd6) [reply packages/cisco_umbrella/data_stream/log/manifest.yml:130]: Others are affected as well. I'll add them to this PR.

**Comment by efd6** [comment]
I have made these changes to obs-cloud-montoring and obs-service-integrations as well since I was here. I hope your toes are OK.

**Review by kcreddy — COMMENTED** [review]
Should we update `aws_logs` package too? Rest LGTM

**Comment by efd6** [comment]
Not at all sure why I missed that one. Done

**Review by kcreddy — APPROVED** [review]
LGTM

**Review by zmoog — APPROVED** [review]
LGTM

**Review by ritalwar — APPROVED** [review]
LGTM

**Pull request merged**